aces/Loris
# [API-docs] Adding static swagger-ui

## Brief summary of changes
This is adding a static version of swagger-ui to LORIS 
see: https://github.com/swagger-api/swagger-ui 

#### Testing instructions (if applicable)

go to `https://<devname>.loris.ca/api_docs`

You should see:
![swagger-ui](https://user-images.githubusercontent.com/12155214/67863415-3cc7b600-fafa-11e9-8375-c875507b313a.png)


To test with different specs, create a `static` directory in the configuration module and download [this file.](https://raw.githubusercontent.com/xlecours/swagger-schemas/configurations/configs.yml) Change the filename to schema.yml. There should be a new entry in the `Select a spec` dropdown.
